lib/telephony/notification_sender.rb

```diff
@@ -0,0 +1,70 @@
+module Telephony
+  class NotificationSender
```

Comment:
A lot of this functionality seems more similar to `AlertSender`, could we move this there to avoid creating a new class?

Reply:
This is specifically as the card requested.

Reply:
ok! that is a surprisingly specific thing for a JIRA to describe

Reply:
hey @dawei-nava, I wrote the original ticket and this was an oversight on my part. I didn't notice the `AlertSender` class when I was researching `Telephony` classes, but looking at it now I agree with Zach that it's a better place for this functionality. I've edited the ticket and moved my original AC into a 'suggested implementation' section.

Reply:
@zachmargolis @tomas-nava moved things around after clarification.